README.md

```diff
@@ -232,6 +233,83 @@ outputs = list(sample['outputs'])
 np.testing.assert_almost_equal(outputs, backend.run_model(model, inputs))
 ```
 
+### Usage - Model quantization
+You can get quantized ONNX models by using [Intel® Neural Compressor](https://github.com/intel/neural-compressor). It provides web-based UI service to make quantization easier and supports code-based usage for more abundant quantization settings.
+
+#### Web-based UI
+You can refer to this [document](https://github.com/intel/neural-compressor/blob/master/docs/bench.md) for how to use web-based UI service. Below is a simple demo:
+![image](https://github.com/onnx/models/blob/main/resource/images/INC_GUI.gif)
```

Comment:
nit: perhaps we can use relative link for the gif file here? In that case, it should be even showable in this PR before merge.

Reply:
> 

Can I keep the gif in README and remove the example code? We want to make it more intuitive

Reply:
Sounds good. That would be much simpler and clearer by only having a gif. Would you still keep the example in another document?

Reply:
Do you have any suggestion for the position of gif, in Usage field or directly below this paragrah '[Intel® Neural Compressor](https://github.com/intel/neural-compressor) is an open-source Python library which supports automatic accuracy-driven tuning strategies to help user quickly find out the best quantized model. It implements dynamic and static quantizations for ONNX models and can represent quantized ONNX models with operator oriented as well as tensor oriented (QDQ) ways. Users can use web-based UI service or python code to do quantization. Read the [Introduction](https://github.com/intel/neural-compressor/blob/master/README.md) for more details.' ?

Reply:
I think keep/not keep the example code are both ok, if you don't mind I can keep it in another document.

Reply:
I prefer to place the gif in the Usage field with a link to another document showing an example code. Thank you!